towr_ros/CMakeLists.txt

```diff
@@ -114,6 +116,12 @@ install(
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
 
+# Mark header files for installation
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  PATTERN ".svn" EXCLUDE
```

Comment:
This should be `PATTERN ".h"`

Reply:
Yes you are right. I was trying to integrate a different robot model to work with towr. For your previous version (where the towr-stack included the packages "towr_core", "towr_examples" and "towr_ros") I just created a separate package with dependencies on towr_ros. This allowed me having a standalone package only with dependencies to yours. However, since you changed the architecture to the current one, I felt it might be necessary to add the robot model directly in a modified towr version, since this allows to switch directly to my robot using your user_interface. 
So when I suggested installing the header files of "towr_ros", I wasn't really aware of the changes you did for the current version.